app/views/shared/_list_description_for_rummager_list.html.erb

```diff
@@ -0,0 +1,27 @@
+<ol class="document-list">
+  <% editions.each_with_index do |edition, edition_index| %>
+    <li class="document-row" id="<%= "#{type}_#{edition.content_id}" %>">
+      <h2>
```

Comment:
If it's possible, using the [document list component](https://govuk-publishing-components.herokuapp.com/component-guide/document_list) here would be ideal, so we can match document lists on other pages and in other applications. 

If not, we should at least remove the `h2` as the document title isn't really a heading in terms of being the heading for a section and it will create a lot of noise for screenreader users, especially those navigating by heading.